cmd/send.go

```diff
@@ -33,8 +33,11 @@ func init() {
 	sendCmd.PersistentFlags().StringVarP(&projectName, "project-name", "p", "", "name of the project to associate test reports with (required)")
 	sendCmd.PersistentFlags().StringVarP(&filePattern, "file-pattern", "f", "", "file name pattern of test reports to send to Fern (required)")
 	sendCmd.PersistentFlags().StringVarP(&tags, "tags", "t", "", "comma-separated tags to be included on runs")
-	sendCmd.MarkPersistentFlagRequired("fern-url")
-	sendCmd.MarkPersistentFlagRequired("project-name")
-	sendCmd.MarkPersistentFlagRequired("file-pattern")
+	err := sendCmd.MarkPersistentFlagRequired("fern-url")
+	err = sendCmd.MarkPersistentFlagRequired("project-name")
+	err = sendCmd.MarkPersistentFlagRequired("file-pattern")
+	if err != nil {
```

Comment:
This will only catch an error from the last call. Errors from the previous 2 lines will be ignored.